### DISCUSSION TITLE
Is it possible to pre-render only the timeline

### DISCUSSION THREAD

**Discussion by kimud6003** [katspaugh/wavesurfer.js#3185] [Q&A] [answered]
I hope to draw the timeline first.
And I want the waveform to be drawn when I add the sound file later.
Is it possible?

**Comment by katspaugh** [comment, marked as answer]
You can load an empty url, empty peaks with an actual duration first to render a timeline. And later load an actual URL.
```js
wavesurfer.load(‘’, [0], 18)
```
